features/admin.users.v1/components/wizard/add-user-wizard.tsx

```diff
@@ -220,6 +230,14 @@ export const AddUserWizard: FunctionComponent<AddUserWizardPropsInterface> = (
         });
     }, [ defaultUserTypeSelection ]);
 
+    /**
+     * Check for workflow associations.
+     */
+    useEffect(() => {
+        if (isWorkflowAssociationDetailsRequestLoading || workflowAssociationDetailsRequestError) return;
+        if (workflowAssociationDetails?.totalResults > 0) setHasWorkflowAssociations(true);
+    }, [ workflowAssociationDetails ]);
```

Comment:
Setting a state variable is not necessary here. We can just derive a local variable and use it.

```
const hasWorkflowAssociations =
  !isWorkflowAssociationDetailsRequestLoading &&
  !workflowAssociationDetailsRequestError &&
  ((workflowAssociationDetails?.totalResults ?? 0) > 0);
```